src/main/resources/integration-jira-log-messages.properties

```diff
@@ -0,0 +1,4 @@
+integration.jira.url.base.invalid=Invalid base URL.
+integration.jira.url.base.invalid.solution=Inform a valid base URL.
+integration.jira.url.authorization.invalid=Invalid authorization callback URL.
```

Comment:
The same comment above, you should change this message to show which callback URL was configured. In the solution message, we should explicitly identify how we can change this callback URL, once it should be configured in the YAML config file.